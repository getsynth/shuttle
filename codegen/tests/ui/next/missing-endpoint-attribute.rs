shuttle_codegen::app! {
    async fn hello() -> &'static str {
        "Hello, World!"
    }

    async fn goodbye() -> &'static str {
        "Goodbye, World!"
    }
}
