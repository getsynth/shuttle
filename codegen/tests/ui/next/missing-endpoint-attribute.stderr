error: missing endpoint attribute

         = help: Try adding `#[shuttle_codegen::endpoint(method = get, route = "/hello")]`

 --> tests/ui/next/missing-endpoint-attribute.rs:2:14
  |
2 |     async fn hello() -> &'static str {
  |              ^^^^^

error: missing endpoint attribute

         = help: Try adding `#[shuttle_codegen::endpoint(method = get, route = "/hello")]`

 --> tests/ui/next/missing-endpoint-attribute.rs:6:14
  |
6 |     async fn goodbye() -> &'static str {
  |              ^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/next/missing-endpoint-attribute.rs:9:2
  |
9 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/next/missing-endpoint-attribute.rs`
